click_repl/__init__.py

```diff
@@ -36,7 +36,7 @@ class ClickExit(RuntimeError):
     text_type = str  # noqa
 
 
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```

Comment:
do we need this now?

Reply:
i swear, there must be a dev branch for these type of changes
so that i can give 0.2.1dev as the version